packages/runtime-dom/src/index.ts

```diff
@@ -63,14 +64,16 @@ export const hydrate = ((...args) => {
 }) as RootHydrateFunction
 
 export const createApp = ((...args) => {
+  const scope = effectScope()
```

Comment:
Should probably be

```suggestion
  const scope = effectScope(true)
```

just in case

Reply:
Done